Tensile/KernelWriterAssembly.py

```diff
@@ -1005,6 +1005,8 @@ def initKernel(self, kernel, tPA, tPB ):
     buffer_load_short = MemoryInstruction("buffer_load_short_d16", 1, 0, 0, 0.5, \
         "UNUSED %s, %s, %s, %s offen offset:0 %s" )
 
+    self.buff_load_inst_offset_max = 4096
```

Comment:
would be good idea to define offset limit variable in memory Instruction and use that here. so in future easier to update